tqdm/tests/tests_tqdm.py

```diff
@@ -1800,3 +1800,18 @@ def test_auto():
     from tqdm import autonotebook, auto
     backendCheck(autonotebook)
     backendCheck(auto)
+
+
+@with_setup(pretest, posttest)
+def test_wrapattr():
+    """Test wrapping file-like objects"""
+    data = "a twenty-char string"
+
+    with closing(StringIO()) as our_file:
+        with closing(StringIO()) as writer:
+            with tqdm.wrapattr(writer, "write", file=our_file) as wrap:
```

Comment:
Should it be called `.wrap_file()`?

Reply:
I was going for

1. something which looks like the built-in naming syntax `getattr`, `setattr`
2. something which wraps generic objects (files, streams, etc) so perhaps `.wrap_obj` or `.wrap_stream`